main_test.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"testing"
+	"time"
+)
+
+func TestRelativeTime(t *testing.T) {
+	tests := []struct {
+		item Item
+		want time.Duration
+	}{
+		{
+			item: Item{
+				// title:  "Random title",
+				// author: "Mr Crabs",
+				// score:  0,
+				absolutePostTime: time.Date(2024, 8, 13, 14, 30, 45, 100, time.UTC),
```

Comment:
Here you should use `hoursSincePosting` and set its value directly.
Using absolute time will not get you to a correct relative time, because calling `time.Since` depends on the current time. So you need to use _only_ relative time.